pkgs/development/tools/pnpm/fetch-deps/default.nix

```diff
@@ -58,9 +63,11 @@
           pnpm config set side-effects-cache false
           # As we pin pnpm versions, we don't really care about updates
           pnpm config set update-notifier false
+          # Run any additional pnpm configuration commands that users provide.
+          ${prePnpmInstall}
```

Comment:
```suggestion
```
see above. this wouldn't be required anymore

Reply:
Another option would be to `runHook prePnpmInstall` here.